web/client/components/misc/enhancers/handleLongTextEnhancer.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright 2023, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import React from "react";
+import OverlayTrigger from "../OverlayTrigger";
+import { Tooltip } from "react-bootstrap";
+
+export  const handleLongTextEnhancer = (RenderFormatter) => (props) => {
```

Comment:
maybe we can call this just Component or Wrapped
```suggestion
export  const handleLongTextEnhancer = (Component) => (props) => {
```